pyproject.toml

```diff
@@ -7,196 +7,150 @@ name = "pymor"
 dynamic = ["version"]
 description = " "
 readme = "README.md"
-license.file = "LICENSE.txt"
+license = {file = "LICENSE.txt"}
```

Comment:
I don't get this change. 'Dotted keys' are part of the official toml spec and the dotted version is much easier to read. (I'm not even sure this is equivalent?)

Reply:
I based it on [this](https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#license).

Reply:
Ok, but why change it? Apparently both versions are equivalent.

Reply:
Ok, if they are equivalent, then I'm fine with either. It is not enforced by `toml-sort`. I reverted the change.